File: src/main/java/com/fendyk/clients/redis/RedisLand.java
package com.fendyk.clients.redis;

import com.fendyk.DTOs.ChunkDTO;
import com.fendyk.DTOs.LandDTO;
import com.fendyk.Main;
import com.fendyk.clients.RedisAPI;
import io.lettuce.core.RedisClient;
import io.lettuce.core.pubsub.RedisPubSubListener;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.UUID;

public class RedisLand extends RedisAPI<LandDTO> {
    public RedisLand(String redisKey) {
        super(redisKey, LandDTO.class);
    }
}

File: src/main/java/com/fendyk/clients/redis/RedisLand.kt
package com.fendyk.clients.redis

import com.fendyk.DTOs.LandDTO
import com.fendyk.clients.RedisAPI
import java.util.concurrent.CompletableFuture

class RedisLand : RedisAPI<String, LandDTO>("land:", LandDTO::class.java) {
    override fun get(key: String): CompletableFuture<LandDTO> {
        return fetch(key)
    }
}
